lib/src/models/chat/chat.dart

```diff
@@ -2,12 +2,21 @@ import 'package:flutter_aira/src/models/chat/chat_feedback_info.dart';
 import 'package:flutter_aira/src/models/conversion_extension.dart';
 
 class ChatSessionInfo {
+  ChatSessionInfo.temp()
```

Comment:
Can I suggest to rename this constructor to `.empty()`? This would be so much more descriptive.